open_event/helpers/oauth.py

```diff
@@ -0,0 +1,22 @@
+class OAuth():
+    """Google Credentials"""
+
+    CLIENT_ID='449612261522-1eg34prt23l0454et59qgqno3rjd8muq.apps.googleusercontent.com'
+    CLIENT_SECRET='aq8XaUlxCfhwwMyZyNw8kS-D'
```

Comment:
We need to think about how we are going to manage CLIENT_SECRET/CLIENT_ID for dev and production use. For dev, we can make it a travis "secret", but that won't work for production.